### ISSUE TITLE
504 error importing XLSX file with field set to "Currency" (?? maybe ??)

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#2504]
**Describe the bug**
Trying to import a relatively simple file, program gives 504 error on matching step

**Expected Behavior**
File should import without issue as far as I can tell

**Actual Behavior**
Program gives 504 error on matching step. Seems like the field set to "Currency" in Excel might be part of the problem (?). See the doc in the folder linked below for a complete description. 

**Steps to Reproduce**
See doc (and files) in this folder
https://drive.google.com/drive/folders/1a83_ML89QIwnA6OkqypohXcHXDWpghxl?usp=sharing

**Instance Information**
Instance: staging (LBNL
SHA: 864f4d8d6

**Additional Context**

**Comment by RDmitchell** [comment]
We need to figure this out relatively soon so that San Jose can get going importing this file into their SEED data.

**Comment by RDmitchell** [comment]
@nllong / @adrian-lara / @axelstudios 

I put this comment into Slack, but am now adding it to this issue, as it is relevant
********
Friday, Jan 8th
I had a call with Hannah at San Jose today, and we successfully uploaded her invoice data (the file referenced in this issue) into her current 2020 cycle on production. However, the progress bar for the final step never got over 0%, and there were no server errors when we opened the developer tools on her browser. We waited about a half hour, and then looked at the Inventory from a different tab, and lo and behold, they had all imported, but no information returned back via the UI that it was successful. Not sure what that means ...

I don't know if this is a result of something that our IT dept did in terms of Cloudflare settings but the situation almost seems worse than it was before.
